materials/tasks.py

```diff
@@ -2,14 +2,21 @@
 from django.core.mail import send_mail
 
 from config.settings import EMAIL_HOST_USER
+from materials.models import Course, SubscribeUpdateCourse
 
 
 @shared_task
-def send_course_update_email(user_email, course_title):
+def send_course_update_email(course_id):
     """Задача для отправки письма подписчикам об обновлении курса"""
-    send_mail(
-        subject=f"Обновление курса: {course_title}",
-        message=f"Курс '{course_title}' был обновлён. Посмотрите новые материалы.",
-        from_email=EMAIL_HOST_USER,
-        recipient_list=[user_email],
-    )
+
+    course = Course.objects.get(id=course_id)
+    subscribers = SubscribeUpdateCourse.objects.filter(course=course, is_subscribe=True)
```

Comment:
nit: так оптимальнее
```suggestion
    subscribers = SubscribeUpdateCourse.objects.select_related('user').filter(course=course, is_subscribe=True)
```

Reply:
Окей, исправлено